code/modules/antagonists/heretic/knowledge/side_void_blade.dm

```diff
@@ -0,0 +1,177 @@
+// Sidepaths for knowledge between Void and Blade.
+
+/// The max health given to Shattered Risen
+#define RISEN_MAX_HEALTH 125
+
+/datum/heretic_knowledge/limited_amount/risen_corpse
+	name = "Shattered Ritual"
```

Comment:
This is ghoul stuff, maybe this will better go to one of the flesh branches instead of void-blade one?

Reply:
The shattered ghoul was inspired by [this ritual](https://cultistsimulator.fandom.com/wiki/Shattered_Risen), which has the aspects "winter" (void) and "edge" (blade).